.github/workflows/update_protos.yml

```diff
@@ -21,7 +21,7 @@ jobs:
         with:
           repo-token: ${{ secrets.GITHUB_TOKEN }}
       - name: Generate buf
-        run: make buf
+        run: mkdir build && cd build && cmake .. -G Ninja -DVIAMCPPSDK_USE_DYNAMIC_PROTOS=ON && ninja update-static-protos
```

Comment:
[nit] it feels like this should be a encapsulated in `make` (or `cmake` ?) command

Reply:
Hmm... I don't think we can easily put all of this in `cmake` since `cmake` is just a way to generate build files that `ninja` is using. As for putting it in a `make` command, I'm a little hesitant just since it's a bit opinionated about the state of the `build` dir (which is part of our `.gitignore` and which we expect users to have). I'm admittedly not an expert on this side of things though, so happy to chat more and be convinced otherwise!

Reply:
Ahh great, thanks! updated :)

Reply:
makes sense - I’m okay leaving as you have it 👌